src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -386,6 +382,17 @@ private function getDefinition(\ArrayObject $definitions, ResourceMetadata $reso
     }
 
     /**
+     * @param string $resourceShortName
+     * @param array  $groups
+     *
+     * @return string
+     */
+    private function getDefinitionKey(string $resourceShortName, array $groups): string
+    {
+        return !$groups ? $resourceShortName : $resourceShortName.'_groups_'.implode('_', $groups);
```

Comment:
I suggest the following to avoid a useless negation: `$groups ? sprintf('%s-%s', $resourceShortName, implode('_', $groups)) : $resourceShortName`

I also suggest to replace the `_groups_` part by a single dash (less verbose).